smarts/core/sumo_traffic_simulation.py

```diff
@@ -426,7 +426,7 @@ def _move_vehicle(self, vehicle_id, position, heading, speed):
             x=x,
             y=y,
             angle=heading,  # only used for visualizing in sumo-gui
-            keepRoute=0b010,  # gives vehicle freedom to move off road
+            keepRoute=0,
```

Comment:
This should be fine for now since we end the episode when vehicles go off route. It could however be a problem now if it's too sensitive.

Reply:
Is there a particular reason for changing this flag? I am not confident in this since we currently have an option to be able to turn off our off-route and I am not sure it plays into our definition of off-route which does not count the oncoming lanes as off-route.

Reply:
If we don't do this, SUMO is unable to bring the vehicle back on the road. We'll have to do some manual vehicle movement to get it back to a region SUMO can pick up where we left off.

Reply:
OK, this may be fine but you will need to check if it works with oncoming lanes or it will disrupt training.

Reply:
Docs seem to suggest =0 should be fine https://sumo.dlr.de/docs/TraCI/Change_Vehicle_State.html#move_to_xy_0xb4

Reply:
According to docs (https://sumo.dlr.de/docs/TraCI/Change_Vehicle_State.html#move_to_xy_0xb4) it seems as though it should be fine.

Reply:
OK, I was confused for a few minutes, but I think the `--lanechange.duration` option is what is allowing this now. Nothing further blocking.